backend/apis/normal-docket-pdf.test.js

```diff
@@ -17,13 +17,15 @@ describe("normal docket pdf file", () => {
     let actual1 = parsedText.charges[1];
 
     expect(actual0.statute).toBe("53-3-202(1)(A)");
-    expect(actual0.description).toBe("NO VALID LICENSE Misdemeanor");
+    expect(actual0.offenseName).toBe("NO VALID LICENSE");
+    expect(actual0.severity).toBe("Misdemeanor");
```

Comment:
This should be `Class C Misdemeanor`

Reply:
Yes I agree it should be I am looking at the pdf right now.  I think the issue is this charge has 3 instances of " - " where normally there are only 2 from what I have seen so far. So we are getting NO VALID LICENSE and then we get back Misdemeanor because it is on the second line
below is what is on the pdf. and we are losing EXPIRED Class C on the split. I could be wrong though.
 Charge 1 - 53-3-202(1)(A) - NO VALID LICENSE - EXPIRED Class C 
         Misdemeanor

Reply:
the dashes aren't how we should be splitting the description string, though - it should be based on uppercase and lowercase words. Right?